### ISSUE TITLE
Chrome deprecation of plan-b semantics

### ISSUE THREAD

**Issue by mychalhackman** [Azure/Communication#263]
**Describe the bug**
As of Chrome 90, a warning appears in the logs for Azure webRTC calls:

```
[Deprecation] Plan B SDP semantics, which is used when constructing an RTCPeerConnection with {sdpSemantics:"plan-b"}, is a legacy version of the Session Description Protocol that has severe compatibility issues on modern browsers. The standardized SDP format, "unified-plan", has been used by default since M72 (January, 2019). Dropping support for Plan B is targeted for M93 (Canary: July 15, 2021; Stable: August 24, 2021).
```
This appears that ACS will be broken on Chrome browsers in the summer if no action is taken?

**To Reproduce**

Begin a call with `callAgent.joinCall()` on Chrome 90+. Sample app should suffice: https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-web

**Expected behavior**
No deprecation warnings in the log regarding "plan-b semantics".

**Screenshots**

![image](https://user-images.githubusercontent.com/25592047/115573148-5ff99e80-a275-11eb-9079-542563b18d60.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 90

**Additional context**
N/A

**Logs**
N/A

**Comment by akania** [comment]
@mychalhackman Thank you for reporting, it's correct, Chrome/Edge plan to deprecate Plan-B and ACS will be migrating to Unified Plan, we'll share dates for when we'll be releasing new versions that will fully support Unified Plan soon

**Comment by mychalhackman** [comment]
Thanks for the good news! Will this also bring Firefox support as well?

**Comment by aboba** [comment]
Please note that as of Chrome M93 (currently in Canary), "Reverse Origin Trial" registration will be required for "Plan B" applications.  Here are some relevant links: 

"Reverse Origin Trial" registration info: https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169

"Plan B" deprecation notice and timeline: https://groups.google.com/g/discuss-webrtc/c/UBtZfawdIAA/m/-UVQQcubBQAJ?pli=1

Chrome feature status: https://www.chromestatus.com/feature/5823036655665152

**Comment by LaithRodan** [comment]
@mychalhackman Firefox support is an item on the ACS backlog, but out so scope for the Unified Plan chrome support.

**Comment by BrentonHolswichQHR** [comment]
Does this mean Microsoft Edge will fail in July/August when 93 comes out?

**Comment by aboba** [comment]
My understanding is that Edge will delay “Plan B” changes for one release (from 93 to 94), to allow customers time to either upgrade to “unified plan” or register in the “reverse origin trial”. 

Jungkee - is the Edge delay confirmed?

**Comment by mychalhackman** [comment]
It's not consistently stated and wasn't obvious for us to find, however v1.1.0 of the JS SDK states it has updated to unified-plan: https://github.com/Azure/Communication/blob/master/releasenotes/acs-javascript-calling-library-release-notes.md#v110-2021-06-17 and this specific quickstart was actually where we noticed the update had been released: https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-web

**Comment by akania** [comment]
@mychalhackman yes, as you've noticed we've released 1.1.0 with UP support - also updated our documentation with [banner](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-web) urging developers to update to that or newer version 
I'll close this issue now
